src/Generator/Formatting.fs

```diff
@@ -297,7 +323,14 @@ let generateStaticClass (functions: PrintReadyTypedFunctionDeclaration[]) (openG
                 |> String.concat ", "
             let funcName = func.prettyName
             // Parameters are so less that running in parallel hurts rather then helps.
```

Comment:
You did remove `Array.Parallel` before, now its back, but the comment remains :)

Reply:
Yeah you got me. Need to change this back as it rather slows down